tests/e2e/cucumber/features/ocm/ocm.feature

```diff
@@ -0,0 +1,20 @@
+@ocm
+Feature: federation management
+
```

Comment:
webUI tests can have a user story

Reply:
so you want me to write steps without step definition